pkg/agent/agent.go

```diff
@@ -84,40 +86,34 @@ func NewAgentOptions() *AgentOptions {
 			// TODO pass them via parameters
 			DecryptionConcurrency: 5,
 			EvaluationConcurrency: 2,
-			EnableMetrics:         true,
```

Comment:
suggest to keep it. We need to support it in future. some policy controllers are using this parameter.